decoder/DecoderApi_unittest.cpp

```diff
@@ -408,6 +456,40 @@ TEST_P(DecodeApiTest, Flush)
     EXPECT_EQ(outFrames, size);
 }
 
+TEST_P(DecodeApiTestLowlatency, Format_Change)
+{
+    SharedPtr<IVideoDecoder> decoder;
+    TestDecodeFrames frames = *GetParam();
+    decoder.reset(createVideoDecoder(frames.getMime()), releaseVideoDecoder);
+    ASSERT_TRUE(bool(decoder));
+
+    VideoConfigBuffer config;
+    memset(&config, 0, sizeof(config));
+    config.enableLowLatency = true;
+    ASSERT_EQ(YAMI_SUCCESS, decoder->start(&config));
+
+    VideoDecodeBuffer buffer;
+    memset(&buffer, 0, sizeof(buffer));
+    FrameInfo info;
+    uint32_t inFrames = 0;
+    uint32_t outFrames = 0;
+    //keep previous resolution
+
+    while (frames.getFrame(buffer, info)) {
+        buffer.flag = 1;
```

Comment:
much better, only one thing here:
please use VIDEO_DECODE_BUFFER_FLAG_FRAME_END instead of [magic numbe](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)